src/integrationCapture.ts

```diff
@@ -0,0 +1,117 @@
+import { SDKEventCustomFlags } from './sdkRuntimeModels';
+import { Dictionary, queryStringParser, getCookies, getHref } from './utils';
+
+
+export const facebookClickIdProcessor = (clickId: string): string => {
```

Comment:
It looks like this processing is always returning the `fbclid` value, but we need it to be returning the formatted `fbc` value - as in of the form `fb.1.1554763741205.AbCdEfGhIjKlMnOpQrStUvWxYz1234567890`.

The `_fbc` value should already be formatted like this so we don't need to process it at all and can use it as is. The `fbclid` value is the last segment (from the example, `AbCdEfGhIjKlMnOpQrStUvWxYz1234567890`). We need to populate the first three segments as described in the docs [here](https://developers.facebook.com/docs/marketing-api/conversions-api/parameters/fbp-and-fbc#2--format-clickid).